project/commons.scala

```diff
@@ -46,6 +46,7 @@ object Commons {
     libraryDependencies ++= Seq(
       "org.specs2" %% "specs2-core" % "3.8.9" % "it,test",
       "org.specs2" %% "specs2-mock" % "3.8.9" % "it,test"
-    )
+    ),
+    exportJars := true
```

Comment:
Without this, we didn't notice the error. This setting makes it so that SBT, rather than using a directory with compiled class files and resources sitting around, will actually compile and use a JAR on the classpath instead.

This matters because it meant when we loaded it as a file, we never noticed it actually fails once it becomes packaged in a JAR. With this setting, that will now never happen again, because our tests work the same way as the real code does once it's compiled.

See http://www.scala-sbt.org/0.13/docs/Howto-Classpaths.html#Use+packaged+jars+on+classpaths+instead+of+class+directories for more info.